packages/rspack/tests/configCases/externals/prefer-provided-over-built-in/webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	target: "node",
+	externals: {
+		http: "1+2"
```

Comment:
This looks quite hacky. But It looks good to me though :)

Reply:
It came from `webpack/test` 🤪